lib/todo/command_menu_formatter.ex

```diff
@@ -0,0 +1,18 @@
+defmodule ToDo.CommandMenuFormatter do
+  def menu_options_for_display do
+    Enum.map_join(commands_title ++ menu_options, "\n", &("#{&1}"))
+  end
+
+  defp commands_title do
+    ["Commands:"]
```

Comment:
What's the need for having this as an additional function and not part of `menu_options` below?